libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/limitValues/shared/limitingItemsHook.ts

```diff
@@ -263,7 +263,7 @@ export const useFilterItems = (
 
             const commonDateFilter: IValuesLimitingItemWithTitle[] = [
                 {
-                    title: filterConfig?.filterName ?? intl.formatMessage({ id: "dateFilterDropdown.title" }),
+                    title: filterConfig?.filterName || intl.formatMessage({ id: "dateFilterDropdown.title" }),
```

Comment:
Actual fix: the filter name was an empty string in the workspace where the issue manifested.